server/src/prisma/schema.prisma

```diff
@@ -0,0 +1,252 @@
+
+  @@index([id], map: "id_1_autocreated")
+  @@map("geojsons")
+}
+
+model ItemCategory {
+  id   String           @id @default(auto()) @map("_id") @db.ObjectId
+  v    Int              @map("__v")
+  name ItemCategoryName
+
+  createdAt DateTime? @default(now())
+  updatedAt DateTime? @updatedAt
+  Item      Item[]
+
+  @@index([name], map: "name_1")
+  @@map("itemcategories")
+}
+
+model Item {
+  id             String   @id @default(auto()) @map("_id") @db.ObjectId
+  v              Int      @map("__v")
+  name           String
+  weight         Float
+  quantity       Int
+  unit           String
+  global         Boolean  @default(false)
+  pack_ids       String[] @db.ObjectId
+  itemCategoryId String?  @db.ObjectId
+
+  createdAt DateTime? @default(now())
+  updatedAt DateTime? @updatedAt
+
+  category ItemCategory? @relation(fields: [itemCategoryId], references: [id])
+  owners   User[]
+  packs    Pack[]        @relation(fields: [pack_ids], references: [id])
+
+  @@index([name], map: "name_1")
+  @@index([weight], map: "weight_1")
+  @@index([unit], map: "unit_1")
+  @@map("items")
+}
+
+model Pack {
+  id                String       @id @default(auto()) @map("_id") @db.ObjectId
+  v                 Int          @map("__v")
+  name              String
+  owner_ids         String[]     @db.ObjectId
+  favoritedById     String?      @db.ObjectId
+  is_public         Boolean?
+  grades            PacksGrades?
+  scores            PacksScores?
+  type              String?      @default("pack")
+  item_ids          String[]     @db.ObjectId
+  favourited_by_ids String[]     @db.ObjectId
+
+  createdAt DateTime? @default(now())
+  updatedAt DateTime? @updatedAt
+
+  favorites_count Int?
+  total_weight    Float?
+  totalScore      Int?
+
+  items       Item[] @relation(fields: [item_ids], references: [id])
+  favoritedBy User[] @relation(name: "Favourites", fields: [favourited_by_ids], references: [id])
+  owners      User[] @relation(name: "Ownership", fields: [owner_ids], references: [id])
+  trips       Trip[]
+
+  @@index([name], map: "name_1")
+  @@index([is_public], map: "is_public_1")
+  @@map("packs")
+}
+
+model Template {
+  id               String       @id @default(auto()) @map("_id") @db.ObjectId
+  v                Int          @map("__v")
+  type             TemplateType
+  templateId       String       @db.ObjectId
+  isGlobalTemplate Boolean      @default(false)
+  createdById      String?      @db.ObjectId
+
+  createdAt DateTime @default(now())
+  updatedAt DateTime @updatedAt
+
+  createdBy User? @relation(fields: [createdById], references: [id])
+
+  @@index([type], map: "type_1")
+  @@index([templateId], map: "templateId_1")
+  @@index([createdById], map: "createdBy_1")
+  @@map("templates")
+}
+
+model Trip {
+  id          String   @id @default(auto()) @map("_id") @db.ObjectId
+  v           Int      @map("__v")
+  name        String
+  // should be JSON?
+  description String
+  // should be JSON?
```

Comment:
Just curious - why do you think these should be JSON?

Reply:
I saw these fields had stringified JSON values in the collection for the provided connection string. So I was just curious why not store JSON instead?